sql/2020/22_HTTP_2/avg_number_of_h2_pushed_resources_and_avg_bytes.sql

```diff
@@ -0,0 +1,26 @@
+#standardSQL
+# 22_HTTP_2 - Average number of HTTP/2 Pushed Resources and Average Bytes
+SELECT 
+  client,
+  COUNT(DISTINCT page) AS num_pages,
+  ROUND(AVG(num_requests),2) AS avg_pushed_requests,
+  ROUND(AVG(kb_transfered),2) AS avg_kb_transfered
+FROM (
+
+SELECT 
+    client,
+    page,
+    SUM(CAST(JSON_EXTRACT_SCALAR(payload, "$._bytesIn") AS INT64)/1024) AS kb_transfered,
+    COUNT(*) AS num_requests
```

Comment:
```suggestion
    COUNT(0) AS num_requests
```

Reply:
Yup. Propagated this change through all of my files